src/exporter/components/export-table/export-table-dialog.tsx

```diff
@@ -141,6 +142,14 @@ export const ExportSatusTableDialog: React.FC<ExportSatusTableDialogProps> = obs
           },
           suppressMovable: true,
         },
+        {
+          headerName: intl.formatMessage({
+            id: 'export-table.table-column-header.sourceLayer.text',
+          }),
+          width: 170,
+          field: 'sourceLayer',
```

Comment:
this code don't belong in this PR - remove it

Reply:
fixed and removed in dbef340